docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,130 @@
+TAG requires incoming connections to be authenticated via host certificates that the host CA issues to Auth and Proxy services.
+
+The host CA can be retrieved and saved into a file in one of the following ways:
+
+<Tabs>
+<TabItem label="Via curl">
+```console
+$ mkdir /etc/access_graph
+$ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=tls-host' > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+
+<TabItem label="Via tctl">
+```console
+$ mkdir /etc/access_graph
+$ tsh login --proxy=<Var name="teleport.example.com" />
+$ tctl get cert_authorities --format=json \
+    | jq -r '.[] | select(.spec.type == "host") | .spec.active_keys.tls[].cert' \
+    | base64 -d > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+</Tabs>
+
+Then, create a configuration file for the TAG service, similar to this:
+
+```yaml
+backend:
+  postgres:
+    # This uses the PostgreSQL connection URI format, see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS
+    # A stricter `sslmode` value is strongly recommended,
+    # e.g. `sslmode=verify-full&sslrootcert=/etc/access_graph/my_postgres_ca.crt`.
+    # For a full reference on possible parameters see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
+    connection: postgres://access_graph_user:mypassword@db.example.com:5432/access_graph_db?sslmode=require
+
+    # When running on Amazon RDS, IAM auth via credentials set in the environment can be used as follows:
+    # iam:
+    #   aws_region: us-west-2
+
+# IP address (optional) and port for the TAG service to listen to.
+# This is the default value. This key can be omitted to listen on port 50051 on all interfaces.
+address: ":50051"
+
+tls:
+  # File paths of PEM-encoded TLS certificate and private key for the TAG server.
+  cert: /etc/access_graph/tls.crt
+  key: /etc/access_graph/tls.key
+
+# This lists the file paths for host CAs of Teleport clusters that are allowed to register with this TAG service.
+# Several paths can be included to allow several Teleport clusters to connect to the TAG service.
+registration_cas:
+  - /etc/access_graph/teleport_host_ca.pem # A full path to the file containing the Teleport cluster's host CA certificate.
+```
+
+Finally, start the TAG service using Docker as follows:
+
+```console
+$ docker run -v <path-to-config>:/app/config.yaml -v /etc/access_graph:/etc/access_graph public.ecr.aws/gravitational/access-graph:(=access_graph.version=)
+```
+
+## Step 2/3. Update Teleport Auth server configuration
+
+In the YAML config for the Auth service, add a new top-level section for Access Graph configuration.
+
+```yaml
+access_graph:
+  enabled: true
+  # host:port where the TAG service is listening
+  endpoint: access-graph.example.com:50051
+  # Optionally, specify a trusted CA we expect the TAG service certificate to be signed by.
+  # If not specified, the system trust store will be used.
```

Comment:
I'd rephrase it. Starting with `optionally` suggests that you can skip this step, which is only true when the TAG's CA is added to a system store.